cmd/ignite/run/create_test.go

```diff
@@ -120,53 +251,28 @@ func TestConstructVMFromCLI(t *testing.T) {
 					PublicKey: "some-pub-key",
 				},
 			},
-			args: []string{testImage},
 			wantSSH: api.SSH{
 				Generate:  true,
 				PublicKey: "some-pub-key",
 			},
 		},
-		{
-			name: "with no VM name and --require-name flag set",
```

Comment:
This was tested twice, check https://github.com/weaveworks/ignite/pull/601/files#diff-d0bf10b7da8bd0e055593597d507f004R306